### PR TITLE
adds support for displaying a scrollbar or not in the suggestion box

### PR DESCRIPTION
**Pull request by miguelpruivo**
As mentioned.

### CODE REVIEW

**Comment by wolfcro1984** [comment]
Can this scrollbar be used to avoid that the suggestion box displays under the keyboard?

**Comment by miguelpruivo** [comment]
> Can this scrollbar be used to avoid that the suggestion box displays under the keyboard?

You can add a restriction to the height of the suggestion box by using the following decoration, for example:
```
suggestionsBoxDecoration: SuggestionsBoxDecoration(
 constraints: BoxConstraints(
   maxHeight: 100.0
 );
```
and this will make a narrow suggestion box with a scrollbar that allows you to go through the suggestions. Is that what you're looking for?

**Comment by wolfcro1984** [comment]
Yes, i think that will be enough. I'll try it when the pull gets accepted. Tnx for the quick answer.

**Comment by AbdulRahmanAlHamali** [comment]
Hey guys, apologies for the delay, I've been a little swamped those few days.

Thank you Miguel for the pull request, I checked it quickly and it seems nice, I will check it closely and accept it over the weekend hopefully. 

Thank you again!

File: lib/flutter_typeahead.dart [hunk 1]
```diff
@@ -955,6 +959,11 @@ class SuggestionsBoxDecoration {
   /// Same as [Material.shape](https://docs.flutter.io/flutter/material/Material/shape.html)
   final ShapeBorder shape;
 
+  /// Defines if a scroll bar will be displayed or not.
+  /// 
+  /// Same as [Scrollbar](https://docs.flutter.io/flutter/material/Scrollbar-class.html)
+  final bool scrollbar;
```
Comment (AbdulRahmanAlHamali) [inline lib/flutter_typeahead.dart]: can we name it hasScrollbar instead?

File: lib/flutter_typeahead.dart [hunk 2]
```diff
@@ -955,6 +959,11 @@ class SuggestionsBoxDecoration {
   /// Same as [Material.shape](https://docs.flutter.io/flutter/material/Material/shape.html)
   final ShapeBorder shape;
 
+  /// Defines if a scroll bar will be displayed or not.
+  /// 
+  /// Same as [Scrollbar](https://docs.flutter.io/flutter/material/Scrollbar-class.html)
```
Comment (AbdulRahmanAlHamali) [inline lib/flutter_typeahead.dart]: No need for this line

File: example/pubspec.lock [hunk 3]
```diff
@@ -87,7 +87,7 @@ packages:
       path: ".."
       relative: true
     source: path
-    version: "0.1.0"
+    version: "0.4.1"
```
[the remark this answers is not in the document]
Comment (miguelpruivo) [reply example/pubspec.lock]: This was a mistake. Should be reverted.

[the remark this answers is not in the document]
Comment (AbdulRahmanAlHamali) [reply example/pubspec.lock]: Sure no worries :) I'm not sure how I can fix the changes on your fork. Do you mind fixing those changes and resubmitting?

[the remark this answers is not in the document]
Comment (miguelpruivo) [reply example/pubspec.lock]: Sure. I’ll fix it tomorrow. :)

[the remark this answers is not in the document]
Comment (AbdulRahmanAlHamali) [reply example/pubspec.lock]: Thank you :)

**Comment by miguelpruivo** [comment]
Hey @AbdulRahmanAlHamali,

I've made the changes you requested. 
Let me know if you need something else.

**Comment by AbdulRahmanAlHamali** [comment]
Merged into develop.

Thank you Miguel. I'm going to make another small fix today, then merge into master and release the new version

**Pull request merged**